principles/index.html

```diff
@@ -64,22 +55,84 @@ <h2>How This Document Fits In</h2>
+
+        <p class="note">
+          Metrics to define significance are being evaluated by a separate task force.
         </p>
-      </div>
+
+        <p>Because measurement and attribution involve all kinds of viewing advertisements and a variety of other actions, in a wide range of different contexts, relying on understanding of, expectations about and consent over cross-context recognition as a result of ad measurement would be inappropriate.</p>
+      </section>
+
+      <section>
+        <h4>Measurement should not significantly enable inferences about individual people from their participation in the measurement</h4>
+
+        <p>Related to cross-context recognition, measurement mechanisms should not reasonably be able to be used to learn or infer information about a particular user, for example, that a user visited a site (or class of site) or took an online or offline action.</p>
+
+        <p>Population-level measurement can still be used for inference; this principle only indicates that participation (or non-participation) in the measurement cannot be used to enable an inference about that individual.</p>
+      </section>
+    </section>
+    <section>
+      <h3>Accountability</h3>
+      <section>
+        <h4>Users should be able to investigate how data about them is used and shared.</h4>
+
+        <p>Users should be able to learn what measurements they participate in.</p>
+        <p>Users should be able to learn what level of risk of re-identification or cross-context data-sharing is possible. 
+          <br><i>See also: comprehensibility.</i></p>
+      </section>
+      <section>
+        <h4>Researchers, regulators and auditors should be able to investigate how a system is used and whether abuse is occurring.</h4>
+
+        <p>Researchers should be able to learn what measurements are taking place, in order to identify unexpected or potentially abusive behavior and to explain the implications of the system to users (whose individual data may not be satisfyingly explanatory).</p>
+
+        <p>Most users will not choose to investigate or be able to interpret individual data about measurements. Independent researchers can provide an important accountability function by identifying potentially significant or privacy-harmful outcomes.</p>
+
+        <p>Some privacy harms -- including to small groups or vulnerable people -- cannot reasonably be identified in the individual case, but only with some aggregate analysis.</p>
+
+        <p>Auditors, with internal access to at least one of the participating systems, should be able to investigate and document whether abuse has occurred (for example, collusion between non-colluding helper parties, or interfering with results). When evidence of abuse is discovered, affected parties must be notified.</p>
```

Comment:
The call today observed that this requirement is not one of our usual strong mathematical ones, but one that is backed by audits.  It might be worth calling that out.